src/ormconfig.ts

```diff
@@ -6,9 +6,10 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 export function OrmConfig(): DynamicModule {
   return TypeOrmModule.forRoot({
     type: 'postgres',
-    synchronize: true,
+    synchronize: false,
```

Comment:
I'm wondering if we should control this based on NODE_ENV (eg true for LOCAL, false otherwise) . Locally it's really nice to have the table created automatically when the service comes up. If we don't do this then we'll have to update a number of our scripts/circle ci configs/readme's to automate/simplify creating their DB tables when testing locally. I agree that in remove env it's safer not to synchronize.

Reply:
This actually does create the able automatically when the service comes up, it just uses the migration files instead of the Entity file.

Reply:
Ah ok - when I tried this locally I didn't have any DB tables created but perhaps I did something wrong...

Reply:
Hm, I'm definitely curious why that might be! Maybe we could screenshare at some point so I can see what's going on? It might be that you need to `npm run build` first. Migrations have to target either the `.ts` or `.js` file or else it'll think there are duplicate migration definitions, so I went with `.js`.